src/Command/ModelCommand.php

```diff
@@ -708,14 +709,12 @@ public function getValidation(Table $model, array $associations, Arguments $args
      * @param \Cake\Database\Schema\TableSchemaInterface $schema The table schema for the current field.
      * @param string $fieldName Name of field to be validated.
      * @param array $metaData metadata for field
-     * @param array<string> $primaryKey The primary key field
      * @return array Array of validation for the field.
      */
     public function fieldValidation(
         TableSchemaInterface $schema,
         string $fieldName,
-        array $metaData,
-        array $primaryKey
+        array $metaData
```

Comment:
We can't change public API without a new major. Just update the description that the param is unused and continue passing the value.

Reply:
Oh right 😅  Will add that back